smart-cards/src/main/java/gov/va/api/health/smartcards/vc/PayloadClaimsWrapper.java

```diff
@@ -0,0 +1,29 @@
+package gov.va.api.health.smartcards.vc;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor
+@JsonAutoDetect(
+    fieldVisibility = JsonAutoDetect.Visibility.ANY,
+    isGetterVisibility = JsonAutoDetect.Visibility.NONE)
+@Schema
```

Comment:
schema tag is unnecessary. (Pretty sure the `@JsonAutoDetect` is mainly used for swagger generation as well.) `@AllArgsConstructor` can be private for good measure.